Lib/functools.py

```diff
@@ -12,7 +12,7 @@
 __all__ = ['update_wrapper', 'wraps', 'WRAPPER_ASSIGNMENTS', 'WRAPPER_UPDATES',
            'total_ordering', 'cmp_to_key', 'lru_cache', 'reduce',
            'TopologicalSorter', 'CycleError',
-           'partial', 'partialmethod', 'singledispatch', 'singledispatchmethod']
+           'partial', 'partialmethod', 'singledispatch', 'singledispatchmethod', "cached_property"]
```

Comment:
Could you move this to the next line so the line remains no longer than ~80 chars?